test/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs

```diff
@@ -67,128 +61,61 @@ public GivenThatWeWantToStoreAProjectWithDependencies(ITestOutputHelper log) : b
+           
+            storeDirectory.Should().HaveFiles(files_on_disk);
         }
 
         [CoreMSBuildOnlyFact]
         public void compose_dependencies_noopt()
         {
             TestAsset simpleDependenciesAsset = _testAssetsManager
-                .CopyTestAsset("SimpleStore")
+                .CopyTestAsset("TargetManifests")
                 .WithSource();
 
-
-            var storeCommand = new ComposeStoreCommand(Log, simpleDependenciesAsset.TestRoot, "SimpleStore.xml");
+            var storeCommand = new ComposeStoreCommand(Log, simpleDependenciesAsset.TestRoot, "FluentAssertion.xml");
 
             var OutputFolder = Path.Combine(simpleDependenciesAsset.TestRoot, "outdir");
             var WorkingDir = Path.Combine(simpleDependenciesAsset.TestRoot, "w");
 
             storeCommand
-                .Execute($"/p:RuntimeIdentifier={_runtimeRid}", $"/p:TargetFramework={_tfm}", $"/p:ComposeDir={OutputFolder}", $"/p:DoNotDecorateComposeDir=true", "/p:SkipOptimization=true", $"/p:ComposeWorkingDir={WorkingDir}", "/p:PreserveComposeWorkingDir=true")
+                .Execute($"/p:RuntimeIdentifier={_runtimeRid}", $"/p:TargetFramework={_tfm}", $"/p:ComposeDir={OutputFolder}", "/p:SkipOptimization=true", $"/p:ComposeWorkingDir={WorkingDir}", "/p:DoNotDecorateComposeDir=true", "/p:PreserveComposeWorkingDir=true", "/p:CreateProfilingSymbols=false")
                 .Should()
                 .Pass();
-
             DirectoryInfo storeDirectory = new DirectoryInfo(OutputFolder);
 
             List<string> files_on_disk = new List<string> {
                "artifact.xml",
-               $"runtime.{_runtimeRid}.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/{_runtimeRid}/native/{_libPrefix}coredistools{FileConstants.DynamicLibSuffix}",
-               $"runtime.{_runtimeRid}.microsoft.netcore.coredistools/1.0.1-prerelease-00001/runtimes/{_runtimeRid}/native/coredistools.h",
-               $"runtime.{_runtimeOs}.system.private.uri/4.4.0-beta-24821-02/runtimes/{_runtimeLibOs}/lib/netstandard1.0/System.Private.Uri.dll"
+               @"newtonsoft.json/9.0.1/lib/netstandard1.0/Newtonsoft.Json.dll",
+               @"fluentassertions/4.12.0/lib/netstandard1.3/FluentAssertions.Core.dll",
+               @"fluentassertions/4.12.0/lib/netstandard1.3/FluentAssertions.dll",
+               @"fluentassertions.json/4.12.0/lib/netstandard1.3/FluentAssertions.Json.dll"
                };
 
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && _testArch != "x86")
-            {
-                files_on_disk.Add($"runtime.{_runtimeRid}.runtime.native.system/4.4.0-beta-24821-02/runtimes/{_runtimeRid}/native/System.Native.a");
-                files_on_disk.Add($"runtime.{_runtimeRid}.runtime.native.system/4.4.0-beta-24821-02/runtimes/{_runtimeRid}/native/System.Native{FileConstants.DynamicLibSuffix}");
-            }
+           
+            storeDirectory.Should().HaveFiles(files_on_disk);
```

Comment:
Then I'd say to account for it in the test.  It is best to be very strict about our outputs, in general.